# Starting point for exploration task, inside the boundary
int32 agent_idx
geometry_msgs/Pose start_pos
nav_msgs/Path input_path
nav_msgs/OccupancyGrid search_map
geometry_msgs/Pose prediction_pos
int32 use_prediction
---
#result definition
nav_msgs/Path output_path
---
#feedback
